TrenDAP/Model/DataSource.cs

```diff
@@ -75,4 +94,45 @@ public class DataSourceTypeController : ModelController<DataSourceType>
         public DataSourceTypeController(IConfiguration configuration) : base(configuration) {} 
     }
 
+    public class DataSourceHelper : XDAAPIHelper
```

Comment:
So this XDAAPIHelper is something we want to use in more than just here (LSCVS and SEBrowser come to mind), so I think it should still live in XDA. That being said, I think a parallel version in .NET core may be wise. Do you know which repo would be the best for that?

Reply:
Maybe you can add it here.
https://github.com/GridProtectionAlliance/openXDA/tree/master/Source/Libraries/openXDA.NetCore

Reply:
Is this something to be addressed in a later commit as well? I think it would be fairly impactful for this PR to move the code into openXDA.NetCore.dll since we can remove all the dependencies on .NET Framework code.